### ISSUE TITLE
Bugged Geodata and following action

### ISSUE THREAD

**Issue by Pavel2451** [L2J/L2J_Server#144]
Hello L2j Team
After long time testing I can say 100% following action when geodata is ON is bugged. Stucking on corners (Chars, pets) some monsters can cast over the wall.

I think following action not using correct geodata

**Comment by Zoey76** [comment]
First issue is cell path finding, second issue is LOS, please check that you are using latest geodata from main site.

**Comment by Pavel2451** [comment]
I using geodata from Maneco

**Comment by Sacrifice78** [comment]
@maneco2 has fixed too many bugs on this geodata... just only need pathengine working better.

**Comment by Pavel2451** [comment]
yes that whyi using it. someone can improve pathnode ngyne? Sometimes it kick me from cata.. or underground from mountain...

**Comment by maneco2** [comment]
@Pavel2451 NOZ geodata is good, geoengine need improvments, not problems geodata files! I only helped a small part for L2J fix the bugs in geoengine

**Comment by Sacrifice78** [comment]
Because @maneco2 is a monsterrrrr :+1

**Comment by Pavel2451** [comment]
SO, when will be first update about engine? I think it's a important..for me too ! :-D

**Comment by maneco2** [comment]
@Pavel2451 I can not say that to you, you need to ask someone from the Staff L2J about engine geo :+1:

**Comment by Sacrifice78** [comment]
me too

**Comment by Pavel2451** [comment]
@Zoey76 Hi, could you improve it for better ? Thanks
